multi-agent/lib/mas/prompts/service.py

```
@@ -0,0 +1,309 @@
+            )
+
+        self._schedule_port.trigger_now(prompt.temporal_schedule_id)
+        return prompt
+
+    def delete(self, prompt_id: str, *, identity: Identity) -> None:
+        prompt = self._load_and_verify(prompt_id, identity)
+
+        if self._schedule_port and prompt.temporal_schedule_id:
+            try:
+                self._schedule_port.delete(prompt.temporal_schedule_id)
+            except Exception:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)